kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -144,9 +145,9 @@
     $trNameSpace: 'management-class-enroll',
     $trs: {
       backToClassDetails: 'Back to class details',
-      enrollSelectedUsers: 'Enroll selected users',
-      selectLearners: 'Select users to enroll in',
-      showingAllUnassigned: 'Showing all users not assigned to this class',
+      enrollSelectedUsers: 'Review Selected Users',
+      selectLearners: 'Choose Users to enroll in',
+      showingAllUnassigned: 'Showing all Users not assigned to this class',
```

Comment:
I'm curious about the choice of capitalization. It seems odd to make this a proper noun?